src/main/scala/PacketAssembler/disassembler.scala

```diff
@@ -31,10 +31,10 @@ object PDAInputBundle {
   * @param done boolean value that indicates the end of current packet
   */
 class PDAOutputBundle extends Bundle {
-  val data = Output(UInt(8.W)) //decouple(sink): data, push, full
-  val length = Output(UInt(8.W))
-  val flag_aa = Output(Valid(Bool()))
-  val flag_crc = Output(Valid(Bool()))
+  val data = Decoupled(UInt(8.W)) //decouple(sink): data, push, full
+  val length = Decoupled(UInt(8.W))
+  val flag_aa = Decoupled(Bool())
```

Comment:
The old DMA uses a single Decouple for flag_aa and flag_crc
Is this interface being used to connect to the old DMA or Alan's new DMA?

Reply:
this is still based on the old DMA. I think Alan haven't changed the new DMA RF side yet

Reply:
What's the more optimal standard? We could just update the new DMA to conform to it.

Reply:
Actually I think the old DMA_RF has separate decouples for flag_aa and flag_crc:
    val paRxFlagAA = Flipped(Decoupled(Bool()))
    val paRxFlagCRC = Flipped(Decoupled(Bool()))

Personally I think separate interfaces make sense because the valid signals of flag_aa and flag_crc are set at different times

Reply:
That's fine, I just wanted to make sure you were aware of the interface while you integrate it into the top level